CONTRIBUTING.md

```diff
@@ -46,12 +45,12 @@ must be given to the long term technical debt. Every new features should
 be covered by functional tests.
 
 When refactoring, structure your PR to make it easy to review and don't
-hesitant to split in multiple small, focused PRs.
+hestitate to split in multiple small, focused PRs.
 
 The Minimal Supported Rust Version is 1.22.0 (enforced by our Travis).
 
-Commit should expose both issues fixed and solutions rational.
-these [guidelines](https://chris.beams.io/posts/git-commit/) should be kept in mind.
+Commit should cover both issues fixed and solutions rationale.
```

Comment:
Probably plural, commits.

Also, solutions is possessive, so either solution's or solutions' (probably the latter if there are multiple solutions, each mapping to an issue fixed.)

Reply:
fixed